seL4-doc/src/appendices/tftpboot_example.md

```diff
@@ -0,0 +1,53 @@
+# Setting up a netboot server using TFTP - Instructions for Linux
+
+The most efficient way of booting from an image is to set up a TFTP server from which the board can read from. This removes the need to repeatedly flash an SD card with a new image, and instead the image can just be copied to the appropriate directory where it can be read and booted. The strategy used will depend on the operating system used, but this guide is for Linux.
+
+1. [Dnsmasq](https://thekelleys.org.uk/dnsmasq/doc.html) provides a network infrastructure for small networks and is designed to be lightweight. The program can be installed using apt-get
+
+```
+sudo apt-get install dnsmasq
+```
+
+2. Create the root folder where the image will be stored. In this example, the path will be `/var/lib/tftpboot`.
+
+2. Identify the ethernet interface you will be connecting the board to using `ip addr` command from the terminal.
+
+![Command `ip addr` output](../figures/ip-addr-ethernet.png)
```

Comment:
I would prefer embedded code listings formatted in markdown rather than images

Reply:
In the image, the ipv6 link-local is not obfuscated, and it's possible to derive the MAC address from it.